lib/paranoia.rb

```diff
@@ -232,6 +234,7 @@ def self.acts_as_paranoid(options={})
       alias_method :destroy_without_paranoia, :destroy
 
       include Paranoia
+      include ParanoiaHelper
```

Comment:
Are we able to `extend ParanoiaHelper`? That way we don't need the fancy `ActiveSupport::Concern`.